_topic_map.yml

```diff
@@ -43,6 +43,13 @@ Topics:
 - Name: Product architecture
   File: acs-architecture
 ---
+Name: Policies and service description
```

Comment:
I don't think "Policies" part makes sense any more. Also, given we have a single topic now, shall we move it somewhere else? Under "Architecture" or under "About" maybe?

Reply:
@rukletsov After discussion with the team we have decided to create a new folder "RHACS Cloud Service" just before the "Architecture" folder.
> RHACS Cloud Service
> |_______Service Description
>                          |______RHACS Cloud Service service description
cc: @kcarmichael08 (for awareness)